.github/composite_actions/launch_ios_simulator/action.yaml

```diff
@@ -3,7 +3,7 @@ description: Launches an iOS simulator and caches it for further action runs
 inputs:
   ios-version:
     description: The iOS version
-    default: "latest"
```

Comment:
do we have to change this back to latest once ios 18 releases? if so, should we put a todo in there?

Reply:
Not necessarily. It's likely that we will have to completely revise our entire process in order to download iOS18 runtimes. I can add a todo to reflect that.